lib/db/keyer.go

```diff
@@ -62,6 +62,9 @@ const (
 
 	// KeyTypeBlockList <block list hash> = BlockList
 	KeyTypeBlockList = 13
+
+	// KeyTypeBlockMapList <int32 folder ID> <block list hash> <file name> = <nothing>
+	KeyTypeBlockMapList = 14
```

Comment:
I'd have expected BlockListMap or BlockListHashMap (or both without Map).

Reply:
So I am open to naming improvements.

The thing that has various hash -> something mappings is called BlockMap, now it  it can also map blocklist hashes to files, so perhaps BlockMapBlockList? 

😱

Reply:
Ups, yeah my names are wrong, these keys aren't directly related to `BlocksList`, just the hash of it. So I assume your `List` refers to the list of filenames? As the `BlockMap` is about individual blocks, I'd definitely use `BlocksHash` in the name to distinguish. Maybe `KeyTypeBlocksHashFiles`?

Reply:
I think it's better now.